PRE-COMMIT.md

```diff
@@ -0,0 +1,12 @@
+# How to use pre-commit
```

Comment:
This wasn't correct on my laptop.  `pip-install pre-commit` did not install a `pre-commit` command; I had to `dnf install pre-commit` to get the pre-commit command.